website/blog/2025-08-07-taotian-practice.md

```diff
@@ -0,0 +1,557 @@
+---
+slug: taotian-practice
+title: "The Implementation Practice Of Fluss On Taotian AB Test Analysis Platform"
```

Comment:
I think the title is a bit long and not convey the important messages, how about this?  cc @polyzos 

- Apache Fluss (Incubating) at Taobao: Powering Real-Time Processing at Scale
- How Taobao uses Apache Fluss (Incubating) for Real-Time Processing in Search and RecSys
- Stream Processing with Apache Fluss (Incubating) in Alibaba Taobao

Reply:
> Apache Fluss (Incubating) at Taobao: Powering Real-Time Processing at Scale
> How Taobao uses Apache Fluss (Incubating) for Real-Time Processing in Search and RecSys

Either of these looks good to me

Reply:
@wuchong for the blog post do we need to have the incubating? Titles get too long and don’t render nicely on the webpage 🥲